doc/developing/misc.md

```diff
@@ -0,0 +1,61 @@
+# Miscellaneous
+
+## Sign the Android APK from CI
+
+The [Github Actions Workflow](https://github.com/minetest/minetest/actions?query=workflow%3Aandroid+event%3Apush)
+automatically produces an APK for each architecture.
+Before installing them onto a device they however need to be signed.
+
+This requires an installation of the Android SDK and `adb`.
+```bash
+.../android-sdk/build-tools/30.0.3/apksigner sign --ks ~/.android/debug.keystore \
```

Comment:
The hardcoded path including what looks like a hardcoded version (30.0.3) seems fragile. On my Ubuntu I can just install `apksigner` (though that may be slightly outdated?).

Reply:
Well you're not supposed to blindly copy-paste it.
`apksigner` appears to be a debian only thing and if you've never used the android sdk then probably the `debug.keystore` doesn't even exist (maybe I should mention that)